### PR TITLE
Updated reset password UX

### PR DESCRIPTION
**Pull request by ryanfeeley**
Improving reset password copy and design so that users understand we
currently wipe their data, so they should have a backup

### CODE REVIEW

**Comment by rfk** [comment]
Looking good.  Will users properly understand the nuance of what "unsaved sync data" means?

Two ways we might try to measure success of this change:

* An overall decrease in the number of resets as a percentage of logins
* An increase in the number of users backing out of the reset page and remembering their password after all

@ryanfeeley do you have any other success measures in mind?

**Comment by ckarlof** [comment]
@ryanfeeley it looks like this is now FXA-72 in Aha.

**Comment by rfk** [comment]
let's do it!  r+

File: features/FxA-72-reset-password/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+
+Reset Password
+==============
+
+https://mozilla.aha.io/features/FXA-72
+
+So that I can avoid inadvertently losing all my Sync data, as a forgetful Firefox Accounts user, I want to understand more clearly what will happen when I reset my password.
+
+## Current/Proposed UX
+![firefox-sync](https://www.lucidchart.com/publicSegments/view/4d816217-8c5a-48be-a14e-79d8eac303d8/image.png)
```
Comment (ckarlof) [inline features/FxA-72-reset-password/README.md]: @ryanfeeley showed this to me yesterday and we tweaked the language a bit so it doesn't say "unsaved sync data" anymore. I feel that linking to external assets in these feature issues somewhat defeats their purpose. If the externally linked assets change, how do the relevant parties learn and acknowledge the change? I felt that the intention was that these feature definitions were self-contained and any changes to them required a PR.

**Pull request merged**